xilem_web/src/events.rs

```diff
@@ -559,8 +565,10 @@ pub struct OnResize<V, State, Action, Callback> {
 
 pub struct OnResizeState<VState> {
     child_state: VState,
-    // reason: Closures are retained so they can be called by environment
-    #[allow(unused)]
+    #[allow(
+        unused,
```

Comment:
Super nit: Using the specific unused lint could be more idiomatic, rather than the entire unused lint group.

In practise, it doesn't really matter.

I'd also maybe like this to be `expect` if it applies in all build configurations.